tests/unit/backend/utils/field-validation/attachment-validation.spec.ts

```diff
@@ -0,0 +1,182 @@
+import { ObjectId } from 'mongodb'
+
+import { ValidateFieldError } from 'src/app/modules/submission/submission.errors'
+import { ProcessedAttachmentResponse } from 'src/app/modules/submission/submission.types'
+import { validateField } from 'src/app/utils/field-validation/'
+import { BasicField, IAttachmentFieldSchema, IField } from 'src/types'
+
+import {
+  generateDefaultField,
+  generateNewAttachmentResponse,
+} from '../../helpers/generate-form-data'
+
+describe('Attachment validation', () => {
+  const makeField = (
+    fieldId: string,
+    attachmentSize: string,
+    customParams?: Partial<IField>,
+  ): IAttachmentFieldSchema => {
+    return {
+      ...generateDefaultField(BasicField.Attachment, customParams),
+      _id: fieldId,
+      attachmentSize,
+    } as IAttachmentFieldSchema
+  }
+
+  const makeResponse = (
+    fieldId: string,
+    content: Buffer | undefined,
+    customParams?: Partial<ProcessedAttachmentResponse>,
+  ): ProcessedAttachmentResponse => {
+    return {
+      ...generateNewAttachmentResponse(customParams),
```

Comment:
I think you can just use `generateNewAttachmentResponse` directly and pass `_id` and `content` in the `customParams`